### ISSUE TITLE
SSL module compilation fails since 2.4.37

### ISSUE THREAD

**Issue by csandanov** [wodby/apache#5]
https://travis-ci.org/wodby/apache/jobs/445998763#L489 

```
httpd: Syntax error on line 159 of /usr/local/apache2/conf/httpd.conf: Cannot load modules/mod_ssl.so into server: Error relocating /usr/local/apache2/modules/mod_ssl.so: SSL_CTX_set_post_handshake_auth: symbol not foun
```

ref https://github.com/docker-library/httpd/issues/115

**Comment by csandanov** [comment]
https://bugs.alpinelinux.org/issues/9741

**Comment by walkero-gr** [comment]
Is this still happening? I am trying to get access to https page on docker, and I get "404 page not found" error

**Comment by csandanov** [comment]
This error has been fixed in Alpine 3.9
